mssql.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/denisenkom/go-mssqldb"
+	"github.com/prometheus/common/log"
+	"strings"
+	"time"
+)
+
+// SQLServerConnection maps a connection to an SQL Server DB
+type SQLServerConnection struct {
+	connectionString string
+	conn             *sql.DB
+}
+
+// StatData values of the statistics data
+type StatData struct {
+	ServerID string
+	Value    string
+	Date     time.Time
+	StatType string
+	Unit     string
+	StatName string
+	ItemName string
+}
+
+// Connect connect to the DB
```

Comment:
Comment could simply be:
```suggestion
// Connect to the DB
```